source/ceylon/time/timezone/parser/iana/parseMonth.ceylon

```diff
@@ -0,0 +1,11 @@
+import ceylon.time.base {
+    months,
+    Month
+}
+
+shared Month parseMonth(String month) {
+    "Invalid Month for parse in timeZone"
+    assert(exists currentMonth = months.all.find((Month elem) 
+        => elem.string.lowercased.startsWith(month.trimmed.lowercased)));
```

Comment:
True.
Given that all month names in the source files are indeed 3 characters long, the problem I outlined is not as severe just as long as these functions are not used outside parsing these rules, so  I guess it's okay.